apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/AssessmentDetailsDialog.tsx

```diff
@@ -645,18 +715,68 @@ const AssessmentDetailsDialog = ({
                         "description of 'skill selection' section of the 'assessment details' dialog",
                     })}
                   </div>
+                  {missingSkills.length ? (
+                    <Well
+                      color="warning"
+                      data-h2-margin-top="base(x.25)"
+                      data-h2-padding="base(x.5)"
+                    >
+                      <p
+                        data-h2-margin-bottom="base(x.5)"
+                        data-h2-font-size="base(caption)"
+                      >
+                        {intl.formatMessage({
+                          defaultMessage:
+                            "The following skills are missing at least 1 assessment",
+                          id: "EY7YQM",
+                          description:
+                            "Warning message for skills with missing assessment on the 'assessment details' dialog",
+                        })}
+                        {intl.formatMessage(commonMessages.dividingColon)}
+                      </p>
+                      <Chips>
+                        {missingSkills.map(({ skill }) => (
+                          <Chip key={skill?.id} color="warning">
+                            {getLocalizedName(skill?.name, intl)}
+                          </Chip>
+                        ))}
+                      </Chips>
+                    </Well>
+                  ) : null}
                 </div>
                 <Checklist
-                  idPrefix="assessedSkills"
-                  id="assessedSkills"
+                  idPrefix="essentialSkills"
+                  id="essentialSkills"
                   name="assessedSkills"
-                  legend={intl.formatMessage(labels.assessedSkills)}
-                  items={assessedSkillsItems}
+                  legend={intl.formatMessage(labels.essentialSkills)}
+                  items={alphaSortOptions(
+                    assessedSkillsItems.essentialSkillItems,
+                  )}
                   rules={{
-                    required: intl.formatMessage(errorMessages.required),
+                    validate: (selectedAssessedSkills: string[]) => {
+                      return selectedAssessedSkills.length > 0;
+                    },
                   }}
                 />
-                {!assessedSkillsItems.length ? (
+                <Checklist
+                  idPrefix="assetSkills"
+                  id="assetSkills"
+                  name="assessedSkills"
+                  legend={intl.formatMessage(labels.assetSkills)}
+                  items={alphaSortOptions(assessedSkillsItems.assetSkills)}
+                  rules={{
+                    validate: (selectedAssessedSkills: string[]) => {
+                      return selectedAssessedSkills.length > 0;
+                    },
+                  }}
+                />
```

Comment:
We will need some sort of control flow here to prevent these checklists from rendering when no items exist:

`assessedSkllsItems.assetSkills.length > 0`

![screenshot_05-Apr-2024_08-25-1712319943](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/b587b95e-7dfd-4ca8-8db1-60fc96d361f4)